src/mugs.js

```diff
@@ -356,7 +356,8 @@ define([
                 .replace("$2", property)
                 .replace("$3", String(vis)));
         },
-        getDisplayName: function (lang) {
+        getDisplayName: function (lang, escape) {
+            if (escape === undefined) { escape = true; }
```

Comment:
I don't think you overwrite `undefined` anymore. Do you think we should stop using `_.isUndefined` in future code?